src/account.ts

```diff
@@ -140,6 +140,13 @@ export class Account {
 
         const flatLogs = [result.transaction_outcome, ...result.receipts_outcome].reduce((acc, it) => acc.concat(it.outcome.logs), []);
         this.printLogs(signedTx.transaction.receiverId, flatLogs);
+        // Find all failures in receipts, print out all except last which is handled later.
+        const receiptFailures = result.receipts_outcome.filter(o => typeof o.outcome.status === 'object' && typeof o.outcome.status.Failure === 'object').map(o => o.outcome.status.Failure);
+        if (receiptFailures.length > 1) {
+            for (let i = 0; i < receiptFailures.length - 1; i++) {
+                console.warn('Receipt failure: ', parseRpcError(receiptFailures[i]));
```

Comment:
Looks like this will result in all errors displayed after all logs (printed by `this.printLogs` above).

This seems like can be confusing. We need to figure out how to print them in order. Looks like it makes sense to log error right after logs for corresponding receipt.

Reply:
okay, pushing up something that groups them together